concepts/05 Widgets/ActionSheet/05 Customize Item Appearance.md

```diff
@@ -11,7 +11,7 @@ For a minor customization of **ActionSheet** buttons, you can define [specific f
         });
```

Comment:
We need make the same changes to this article as we did, for example, to the [TileView article](https://github.com/DevExpress/devextreme-documentation/pull/413/files#diff-9d7a06ab9bff92e89bc374ff40ad26d4R1).

Reply:
fixed